frontend/src/lib/voter/vaa-filters/src/group/combineResults.ts

```diff
@@ -3,12 +3,14 @@ import type { MaybeWrapped } from '../entity';
 /**
  * A logic operator for combining results from multiple filters.
  */
-export enum LogicOp {
+export const LogicOp: Record<'And' | 'Or', number> = {
```

Comment:
Ditto: camel case and typing can likely be removed.

Reply:
Typing is neede